index.ts

```diff
@@ -1 +1 @@
-export * as default from './fabric';
+export * from './fabric';
```

Comment:
if we want the umd module to keep working my understanding is that this needs to be an export default.
Does this change fix something broken?

Reply:
There is no need for a default export. It is wrong.
Rollup handles the naming in the umd build according to the `name` field.
So this allows all build types including es, cjs

Reply:
i didn't try this yet, if it doesn't work as planned, we can still put back the default. but as it is, index.ts is a duplicate of fabric not usefult anymore.

Reply:
Also this wasn't here for the naming, but becuase some tool explicitly required a default export.
The name is gone since we don't do anymore `import { fabric } from 'fabric'`

Reply:
no sorry, name is because window.fabric. but is still unrelated to the default export.